A6-client-retries.md

```diff
@@ -402,10 +398,10 @@ The retry policy is transmitted to the client through the service config mechani
       // RPCs will not be retried or hedged.
 
       "retryPolicy": {
-        // The maximum number of retry attempts, exclusive of the original RPC.
+        // The maximum number of RPC attempts, including the original RPC.
         //
-        // This field is required and must be greater than zero.
-        "maxRetryAttempts": number,
+        // This field is required and must be one or greater.
```

Comment:
It should actually be 2 or greater.  Given the new semantic, if it's set to 1, it means that no retrying or hedging should be done, so there's no point in supporting that value.